content/tutorials/build-apps/build-a-desktop-wallet-in-javascript.md

```diff
@@ -0,0 +1,1460 @@
+          <small class="text-muted">- Part 6/8</small>
+        </h3>
+      </div>
+
+      <div class="tab-pane fade show active" id="dashboard" role="tabpanel" aria-labelledby="dashboard-tab">
+        <h3>Account:</h3>
+        <ul class="list-group">
+          <li class="list-group-item">Classic Address: <strong id="account-address-classic"></strong></li>
+          <li class="list-group-item">X-Address: <strong id="account-address-x"></strong></li>
+          <li class="list-group-item">XRP Balance: <strong id="account-balance"></strong></li>
+          <li class="list-group-item">XRP Reserved: <strong id="account-reserve"></strong></li>
+        </ul>
+        <div class="spacer"></div>
+        <h3>
+          Ledger
+          <small class="text-muted">(Latest validated ledger)</small>
+        </h3>
+        <ul class="list-group">
+          <li class="list-group-item">Ledger Index: <strong id="ledger-index"></strong></li>
+          <li class="list-group-item">Ledger Hash: <strong id="ledger-hash"></strong></li>
+          <li class="list-group-item">Close Time: <strong id="ledger-close-time"></strong></li>
+        </ul>
+      </div>
+
+      <div class="tab-pane fade" id="transactions" role="tabpanel" aria-labelledby="transactions-tab">
+        <h3>Transactions:</h3>
+        <table id="tx-table" class="table">
+          <thead>
+          <tr>
+            <th>Confirmed</th>
+            <th>Type</th>
+            <th>From</th>
+            <th>To</th>
+            <th>Value Delivered</th>
+            <th>Hash</th>
+          </tr>
+          </thead>
+          <tbody></tbody>
+        </table>
+      </div>
+
+    </div>
+
+  </main>
+
+  <dialog id="seed-dialog">
+    <form method="dialog">
+      <div>
+        <label for="seed-input">Enter seed:</label>
+        <input type="text" id="seed-input" name="seed-input" />
+      </div>
+      <div>
+        <button type="submit">Confirm</button>
+      </div>
+    </form>
+  </dialog>
+
+  <dialog id="password-dialog">
+    <form method="dialog">
+      <div>
+        <label for="password-input">Enter password (min-length: 5):</label>
+        <input type="text" id="password-input" name="password-input" />
+      </div>
+      <div>
+        <button type="submit">Confirm</button>
+      </div>
+    </form>
+  </dialog>
+
+</body>
+
+<script src="../bootstrap/bootstrap.bundle.min.js"></script>
+<script src="renderer.js"></script>
+
+</html>
+````
+
+Here we basically added the [Boostrap Framework]() and a little custom styling to our application. We'll leave it at that for this Step - to get the application running at this stage of development, run the following command:
+
+```console
+npm run start
+```
+
+This time the application should display a dashboard like layout with the XRP logo and a navigation ono the left side, and a content area comprising most of the screen. You should be able to switch from the dashboard to the transactions list and back.
```

Comment:
For some reason, when I run this, I get an error which causes the "Seed" / "Password" prompt to not appear.

```
Uncaught Error: Cannot read properties of null (reading 'addEventListener')
    at EventEmitter.emit (VM4 sandbox_bundle:2:35718)
    at Object.onMessage (VM4 sandbox_bundle:2:47921)
```

<img width="1024" alt="image" src="https://github.com/XRPLF/xrpl-dev-portal/assets/19694186/2a89b4e6-1b59-4185-9779-ef3b9ed400de">

Reply:
[Archive.zip](https://github.com/XRPLF/xrpl-dev-portal/files/12255084/Archive.zip)

Here's my code (You'll need to `npm i`)

Reply:
I think this is caused by the missing button for `change seed` - re-adding that fixes the error.